src/current/_includes/v23.1/misc/enterprise-features.md

```diff
@@ -1,12 +1,24 @@
-[Single Sign-on (SSO) for DB Console](sso-db-console.html) | This feature allows you to grant access to DB Console to identities managed in an external identity provider (IdP).
-[Cluster Single Sign-on (SSO)](sso-sql.html) | This feature allows you to grant SQL access to {{ site.data.products.core }} clusters to identities managed in an external IdP.
\ No newline at end of file
+[Follower Reads](follower-reads.html) | Reduce read latency in multi-region deployments by using the closest replica at the expense of reading slightly historical data.
+[Multi-Region Capabilities](multiregion-overview.html) | Row-level controls where your data is stored to help you reduce read and write latency and meet regulatory requirements.
+[Node Map](enable-node-map.html) | Visualize the geographical distribution of a cluster by plotting its node localities on a world map.
+
+## Recovery and streaming
+
+Feature | Description
+--------+-------------------------
+Enterprise [`BACKUP`](backup.html) and restore capabilities | Take and restore [Incremental backups](take-full-and-incremental-backups.html), [backups with revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), [locality-aware backups](take-and-restore-locality-aware-backups.html), and [encrypted backups](take-and-restore-encrypted-backups.html) require an Enterprise license. [Full backups](take-full-and-incremental-backups.html) do not require an Enterprise license.
+[Changefeeds into a Configurable Sink](create-changefeed.html) | For every change in a configurable allowlist of tables, configure a changefeed to emit a record to a configurable sink, either Apache Kafka or a cloud-storage sink. These records can be processed by downstream systems for reporting, caching, or full-text indexing.
+[Change Data Capture Queries](cdc-queries.html) | Use `SELECT` queries to filter and modify change data before sending it to a changefeed's sink.
+
+## Security and IAM
+
+Feature | Description
+--------+-------------------------
+[Encryption at Rest](security-reference/encryption.html#encryption-at-rest-enterprise) | Enable automatic transparent encryption of a node's data on the local disk using AES in counter mode, with all key sizes allowed. This feature works together with CockroachDB's automatic encryption of data in transit.
+[GSSAPI with Kerberos Authentication](gssapi_authentication.html) | Authenticate to your cluster using identities stored in an external enterprise directory system that supports Kerberos, such as Active Directory.
+[Cluster Single Sign-on (SSO)](sso-sql.html) | Grant SQL access to {{ site.data.products.core }} clusters to identities managed in an external IdP.
+[Single Sign-on (SSO) for DB Console](sso-db-console.html) | Grant access to a cluster's DB Console interface to identities managed in an external identity provider (IdP).
```

Comment:
suggest switching  the IdP definitions here so the acronym is defined in the Cluster SSO description and used afterwards in the DB Console SSO description